packages/lib-user/src/components/UserStats/UserStats.js

```diff
@@ -42,18 +43,26 @@ function UserStats({
   // set top projects based on selected date range and all project stats
   let topProjects = []
   const topProjectContributions = allProjectsStats.project_contributions
-    .sort((a, b) => b.count - a.count)
+    ?.sort((a, b) => b.count - a.count)
 
   topProjects = topProjectContributions
-    .map(projectContribution => {
+    ?.map(projectContribution => {
       const projectData = projects?.find(project => project.id === projectContribution.project_id.toString())
       return projectData
     })
     .filter(project => project)
     .slice(0, 5)
 
   return (
-    <Layout>
+    <Layout
+      primaryHeaderItem={
+        <HeaderLink
+          href={`/users/${user?.login}`}
```

Comment:
When clicking this link, a user is "going back" to PFE, meaning they're leaving app-root. I think this has to be hardcoded as `https://www.zooniverse.org/users/...` in order to hit the static repo and route a user correctly. Otherwise, a Next.js Link component in app-root will think it's okay to route to the file in `app-root/src/app/users/page.js`, which is empty.